docs/e2e.md

```diff
@@ -0,0 +1,13 @@
+# End-to-end testing with Cypress
+
+## Run Locally
+
+* `npm run build`
```

Comment:
I think this section should be something along the lines of

- Run the website locally with `npm start`
- Live run the test suite with `npm run cypress:open`
- Run the tests in a headless environment with `npm run cypress:cli`
- Travis will invoke `npm run cypress:travis` to run the tests.